java/midp/com/sun/midp/l10n/LocalizedStringsBase.java

```diff
@@ -1,3 +1,29 @@
+/*
```

Comment:
> Doesn't java/midp/LICENSE apply to this file as well?

Yes, but all the rest of the files also have their own license headers, which all say that "You should have received a copy of the GNU General Public License along with this program". Thus I'm adding a copy of the license itself in the LICENSE file.